Gemfile.lock

```diff
@@ -10,22 +10,22 @@ GEM
   remote: https://rubygems.org/
   specs:
     eventmachine (1.2.7)
-    faye-websocket (0.10.7)
+    faye-websocket (0.10.9)
       eventmachine (>= 0.12.0)
       websocket-driver (>= 0.5.1)
     mustermann (1.0.3)
-    rack (2.0.7)
-    rack-protection (2.0.5)
+    rack (2.0.8)
+    rack-protection (2.0.7)
       rack
-    sinatra (2.0.5)
+    sinatra (2.0.7)
```

Comment:
### [sinatra](http://sinatrarb.com/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.5 → 2.0.7

[_[change-log](https://github.com/sinatra/sinatra/blob/master/CHANGELOG.md), [source-code](https://github.com/sinatra/sinatra)_]





<details>
<summary>Commits</summary>

A change of **49** commits. See the full changes on [the compare page](https://github.com/sinatra/sinatra/compare/v2.0.5...v2.0.7).

These are the first 10 commits:
- (7b429cb) [CI matrix: add 2.6.0, update JRuby to 9.2.5.0](https://github.com/sinatra/sinatra/commit/7b429cb0e3f1cf09c34e5f8a408397f19b9b10e9)
- (f77a0d0) [Merge pull request #1511 from olleolleolle/patch-4](https://github.com/sinatra/sinatra/commit/f77a0d0baf62c634d089cbc2f0df60049002e247)
- (ea3f43d) [Handle ruby-2.2.10 version on travis builds](https://github.com/sinatra/sinatra/commit/ea3f43ddabfb6796e84c864bcfd33bae9ecd5215)
- (8b693bf) [Update Rack::Protection link in Readme.*.md](https://github.com/sinatra/sinatra/commit/8b693bfac667f1a164fca7e6f5b75f206752976d)
- (26fa5c8) [Merge pull request #1516 from yuki3738/update-rack-protectio…](https://github.com/sinatra/sinatra/commit/26fa5c86220890a02cc0f070ddb2696d74a5b843)
- (d8c1839) [Support nil param on Ruby < 2.4](https://github.com/sinatra/sinatra/commit/d8c18394d86626726c256ce57c3336ca649a33ae)
- (cb4f651) [Merge pull request #1513 from gowda/travis-ruby-2.2.10](https://github.com/sinatra/sinatra/commit/cb4f651282295803e7484a5841f6285b87e85228)
- (932665f) [ignore bundler/inline from callers](https://github.com/sinatra/sinatra/commit/932665fe159d82b47ee682d2a92597ed3f402fac)
- (4ceed83) [Merge pull request #1520 from sinatra/fix-1507](https://github.com/sinatra/sinatra/commit/4ceed83a16ea3f3b018d11004079393baa5c5200)
- (2e36823) [Merge pull request #1517 from segiddins/segiddins/nil-param](https://github.com/sinatra/sinatra/commit/2e368238238358538cc0e02fe900f3ea05705c0a)


</details>